plutus-playground-client/src/MonadApp.purs

```diff
@@ -121,14 +120,14 @@ runAjax ::
   HalogenApp m (WebData a)
 runAjax action = wrap $ RemoteData.fromEither <$> runExceptT action
 
-withEditor :: forall a m. MonadEffect m => (Editor -> Effect a) -> HalogenApp m (Maybe a)
-withEditor = HalogenApp <<< Editor.withEditor _editorSlot unit
+withEditor :: forall a m. MonadEffect m => (Editor -> m a) -> HalogenApp m (Maybe a)
+withEditor f = HalogenApp $ Editor.withEditor _editorSlot unit f
```

Comment:
presumably this is necessary after changing type of `f`?

Reply:
Ah, good spot. It's just a hangover from me reworking the code. I'll revert it so we've just got the type signature change. Thanks.